ui/src/app.tsx

```diff
@@ -296,35 +298,27 @@ function GroupsRoutes({ state, location, isMobile, isSmall }: RoutesProps) {
               element={
                 <Notifications
                   child={GroupNotification}
-                  title={`Activity • ${appHead('').title}`}
+                  title={`Activity • ${groupsTitle}`}
                 />
               }
             />
             {/* Find by Invite URL */}
             <Route
               path="/find/:ship/:name"
-              element={
-                <FindGroups title={`Find Groups • ${appHead('').title}`} />
-              }
+              element={<FindGroups title={`Find Groups • ${groupsTitle}`} />}
             />
             {/* Find by Nickname or @p */}
             <Route
               path="/find/:ship"
-              element={
-                <FindGroups title={`Find Groups • ${appHead('').title}`} />
-              }
+              element={<FindGroups title={`Find Groups • ${groupsTitle}`} />}
             />
             <Route
               path="/find"
-              element={
-                <FindGroups title={`Find Groups • ${appHead('').title}`} />
-              }
+              element={<FindGroups title={`Find Groups • ${groupsTitle}`} />}
```

Comment:
Let's switch from 'Find Groups' to 'Discover'. A bit confusing, haven't renamed all the components quite yet.